src/renderer/views/SubscribedChannels/SubscribedChannels.vue

```diff
@@ -53,7 +53,7 @@
               class="unsubscribeContainer"
             >
               <ft-subscribe-button
-                class="btn"
+                :hide-profile-dropdown-toggle="false"
```

Comment:
As the default value for that property is false, you can probably remove this line.